places/scalable/distributor-logo-archlinux.svg

```diff
@@ -1,107 +1,15 @@
-        color:#3daee9;
-      }
-      .ColorScheme-ButtonText {
-        color:#31363b;
-      }
-      .ColorScheme-ButtonBackground{
-        color:#eff0f1;
-      }
-      .ColorScheme-ButtonHover {
-        color:#3daee9;
-      }
-      .ColorScheme-ButtonFocus{
-        color:#3daee9;
-      }
-	</style>
-    <linearGradient id="bg" x1="0" y1="1" x2="0" y2="0">
-      <stop stop-color="#0a80fd" offset="0" />
-      <stop stop-color="#56c5fa" offset="1" />
-    </linearGradient>
-  </defs>
-    <circle cx="24" cy="24" r="22" fill="url(#bg)" />
-	<path d="m 24.215798,9.9669553 c -1.13723,2.7881197 -1.82311,4.6118497 -3.08922,7.3168197 0.77625,0.82289 1.72912,1.78114 3.27662,2.86344 -1.66368,-0.68463 -2.79847,-1.37192 -3.64653,-2.08513 -1.62037,3.38126 -4.15913,8.19766 -9.31099,17.45434 4.0492,-2.33762 7.1878,-3.77881 10.11325,-4.32878 -0.1256,-0.54023 -0.19705,-1.12463 -0.19219,-1.73436 l 0.003,-0.12974 c 0.0643,-2.59421 1.41372,-4.58912 3.01231,-4.45365 1.5986,0.1354 2.8412,2.34949 2.77695,4.9437 -0.012,0.4882 -0.0672,0.95772 -0.16333,1.39329 2.8935,0.56604 5.99877,2.00353 9.99315,4.30954 -0.78762,-1.45002 -1.49063,-2.75716 -2.16195,-4.00206 -1.05752,-0.81962 -2.1605,-1.88632 -4.41042,-3.04118 1.54648,0.40184 2.65373,0.86544 3.51681,1.38366 -6.82563,-12.70848 -7.37814,-14.39732 -9.71914,-19.8905397 z" fill="#FFF" opacity="0.85" />
+<svg height="48" width="48" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <linearGradient id="a" gradientUnits="userSpaceOnUse" x1="399.57144" x2="399.57144" y1="545.79797" y2="517.79797">
+    <stop offset="0" stop-color="#3889e9" />
+    <stop offset="1" stop-color="#5ea5fb" />
+  </linearGradient>
+  <linearGradient id="b" gradientUnits="userSpaceOnUse" x1="399.9939" x2="399.9939" y1="545.13873" y2="518.13873">
+    <stop offset="0" stop-color="#0a80fd" />
+    <stop offset="1" stop-color="#56c5fa" />
+  </linearGradient>
+  <g stroke-width="1.299897" transform="matrix(1.7142909 0 0 1.7142909 -662.715871 -887.647323)">
+    <circle cx="400.57138" cy="531.79791" fill="url(#b)" r="13.999958" />
+    <circle cx="400.57138" cy="531.79791" fill="url(#a)" r="0" />
+  </g>
+  <path d="m673.99096 786.06661c-1.20204 2.94701-1.92701 4.87468-3.26527 7.73381.82049.86979 1.82767 1.88264 3.46336 3.02662-1.75849-.72364-2.95796-1.4501-3.85435-2.20396-1.71272 3.57396-4.39615 8.66484-9.84162 18.44905 4.27996-2.47084 7.59744-3.99415 10.6896-4.57546-.13276-.57103-.20827-1.18872-.20314-1.83321l.004-.13713c.0679-2.74205 1.4943-4.85066 3.18399-4.70747 1.6897.14313 3.00312 2.48339 2.9352 5.22544-.0127.51602-.071 1.01231-.17264 1.4727 3.05841.5983 6.34064 2.11771 10.56267 4.55513-.83252-1.53265-1.57559-2.91428-2.28517-4.23012-1.11779-.86634-2.28362-1.99383-4.66176-3.21451 1.63461.42475 2.80496.91477 3.71723 1.46252-7.21462-13.43273-7.79862-15.21782-10.27303-21.02409z" fill="#fff" fill-opacity=".85" fill-rule="evenodd" transform="matrix(1.03209074 0 0 1.03209074 -671.404402 -802.591814)" />
```

Comment:
Am I crazy, or has the icon size been increased here? The rich diff seems to show a lack of padding between the image border and the icon.

... If I'm not crazy, please resize this icon back as it was before.

Reply:
Okay I'm not crazy! This also happened for the Debian, Fedora, Gentoo, Mageia, Manjaro, OpenSUSE, Rosa, and Ubuntu distributor logos.